napari/_qt/qt_dims.py

```diff
@@ -1,5 +1,11 @@
 from qtpy.QtCore import Qt, Signal
-from qtpy.QtWidgets import QWidget, QGridLayout, QSizePolicy, QScrollBar
+from qtpy.QtWidgets import (
+    QWidget,
+    QGridLayout,
+    QSizePolicy,
+    QScrollBar,
+    QLabel,
```

Comment:
I think we want to make this QLineEdits so that you could edit the labels if you want. You can see examples of how we've done text edit boxes inside our layer widget - here https://github.com/napari/napari/blob/708271c47549e9de9b63316dfec6e381fcb0c7d2/napari/_qt/qt_layerlist.py#L344
you'll have to propogate any changes in the text back to the `axis_labels` property on the `Dims` model, but that shouldn't be too bad

Reply:
One of the advantages of the QLineEdit is that we can give it a fixed width and you then get scrolling left / right for free if someone uses a really long name